### PR TITLE
Work section, about me And contact form.

### PR DESCRIPTION
**Pull request by christianbiring1**
Here I had to Add 

- [ ] The work section of my portfolio project
- [ ]  About me, and
- [ ]  contact form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,6 +72,186 @@ <h1 class="primary-text">I'm Christian<br />Glad to see<br />you!</h1>
...
+            sign-ups required.
+          </div>
+          <div class="languages">
+            <ul>
+              <li><span class="html">HTML</span></li>
+              <li><span class="css">CSS</span></li>
+              <li><span class="js">JavaScript</span></li>
+            </ul>
+          </div>
+          <a href="#">See project</a>
```
[the remark this answers is not in the document]
Comment (christianbiring1) [reply index.html]: everything fixed about buttons

File: index.html [hunk 2]
```diff
@@ -68,6 +72,186 @@ <h1 class="primary-text">I'm Christian<br />Glad to see<br />you!</h1>
...
+                ><img src="images/javascript.png" alt="js-icone" />
+                <span class="label">JavaScript</span> </span
+              ><br />
+              <span class="icone"
+                ><img src="images/html.png" alt="html-icone" />
+                <span class="label">HTML</span></span
+              ><br />
+              <span class="icone"
+                ><img src="images/css.png" alt="" />
+                <span class="label">CSS</span></span
```
[the remark this answers is not in the document]
Comment (christianbiring1) [reply index.html]: - [ ] All my categories are in a li I don't know how you didn't see that. 
- [ ] All the margin, padding ... values are now fixed. I'm good now.

File: index.html [hunk 3]
```diff
@@ -68,6 +72,186 @@ <h1 class="primary-text">I'm Christian<br />Glad to see<br />you!</h1>
...
+      <section class="about">
+        <h1>About<br />Myself</h1>
+        <p class="content">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hestiate to
+          contact me.
+        </p>
+        <div class="social">
+          <p>let's connect</p>
```
[the remark this answers is not in the document]
Comment (sja-thedude) [reply index.html:178]: - [x] As asked by the previous reviewer, kindly change `<p>` tag on `lets connect` to `<h3>` or `<h4>`.

File: index.html [hunk 4]
```diff
@@ -68,6 +72,189 @@ <h1 class="primary-text">I'm Christian<br />Glad to see<br />you!</h1>
...
+                <span class="label">JavaScript</span>
+              </span>
+              <span class="icone"
+                ><img src="images/html.png" alt="html-icone" />
+                <span class="label">HTML</span></span
+              >
+              <span class="icone"
+                ><img src="images/css.png" alt="" />
+                <span class="label">CSS</span></span
+              >
```
Comment (sja-thedude) [inline index.html]: - [ ] As asked by the previous reviewer, You have not used the list tags in these. You have to use the `ul` tag for languages and the `li` tag for all the elements Javascript, HTML, CSS. Fix it.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
Hi @christianbiring1, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- All linters are passing 👍 
- Professional Readme 👍 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (christianbiring1) [reply index.html:178]: He didn't even say here
 she actually said to use the div I was putting on the main text with the class description in my HTML.
this let's connect is not even something to highlight here. it's just a simple paragraph.

File: index.html (hunk 4 above)
Comment (christianbiring1) [reply index.html]: is that let's connect a title? my review for the first project was actually the one who told me to put that p right there

**Review by sja-thedude — APPROVED** [review]
Hi @christianbiring1,  This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**